src/Service/OnSiteDetailsReportingService.php

```diff
@@ -14,29 +17,94 @@ class OnSiteDetailsReportingService
         'Imported'
     ];
 
+    public static $incentiveExportHeaders = [
+        'Date Created',
+        'Participant ID',
+        'User',
+        'Date of Service',
+        'Occurrence',
+        'Type',
+        'Amount',
+        'Declined?',
+        'Notes',
+        'Imported',
+        'Amended'
+    ];
+
     public static $patientStatusSortColumns = [
         'psh.createdTs',
         'ps.participantId',
         'u.email',
-        'psh.site',
+        's.name',
         'psh.status',
         'psh.comments'
     ];
 
-    public function getAjaxData($patientStatuses): array
+    public static $incentiveSortColumns = [
+        'i.createdTs',
+        'i.participantId',
+        'u.email',
+        'i.incentiveDateGiven',
+        'i.incentiveOccurrence',
+        'i.incentiveType',
+        'i.incentiveAmount',
+        'i.declined',
+        'i.notes'
+    ];
+
+    public function getPatientStatusAjaxData($patientStatuses): array
     {
         $rows = [];
         foreach ($patientStatuses as $patientStatus) {
             $row = [];
             $row['created'] = $patientStatus['createdTs']->format('m-d-Y');
             $row['participantId'] = $patientStatus['participantId'];
             $row['user'] = $patientStatus['email'];
-            $row['site'] = $patientStatus['site'];
-            $row['patientStatus'] = $patientStatus['status'];
+            $row['site'] = $patientStatus['siteName'];
+            $row['patientStatus'] = array_search($patientStatus['status'], PatientStatus::$onSitePatientStatus);
             $row['notes'] = $patientStatus['comments'];
             $row['importId'] = $patientStatus['importId'] ? 'Yes' : 'No';
             array_push($rows, $row);
         }
         return $rows;
     }
+
+    public function getIncentiveTrackingAjaxData($incentives, $export = false): array
+    {
+        $rows = [];
+        foreach ($incentives as $incentive) {
+            $row = [];
+            $row['created'] = $incentive['createdTs']->format('m-d-Y');
+            $row['participantId'] = $incentive['participantId'];
+            $row['user'] = $incentive['email'];
+            $row['dateOfService'] = $incentive['incentiveDateGiven']->format('m-d-Y');
+            $row['occurrence'] = $incentive['incentiveOccurrence'] === 'other' ? 'Other, ' . $incentive['otherIncentiveOccurrence'] : array_search($incentive['incentiveOccurrence'], Incentive::$incentiveOccurrenceChoices);
+            $type = array_search($incentive['incentiveType'], Incentive::$incentiveTypeChoices);
+            if ($type === 'other') {
+                $type = 'Other, ' . $incentive['otherIncentiveType'];
+            } elseif ($type === 'gift_card') {
```

Comment:
There are a number of places where we are checking to see if a type equals a string (`gift_card`, `other`, etc.).  Instead of strings, could we use constants?